.travis.yml

```diff
@@ -0,0 +1,22 @@
+language: python
+sudo: false
+
+addons:
+  apt:
+    packages:
+      - rpm
+
+python:
+  #- 2.7
```

Comment:
Python 2.7 used to work I guess. Haven't tested it in a while, though.
Maybe we can give it a shot?

Reply:
yep, it can build, but the test cases can't run at all. It fails at `from json import JSONDecodeError`.